stories/components/FilterMultiSelect.js

```diff
@@ -59,9 +59,11 @@ export default class FilterMultiSelect extends React.Component {
   }
 
   getSelectValue = (props) => {
+    const { allOption, allowSelectAll, options } = props
```

Comment:
Is this case, since we want to raw unfiltered options, can we use `const { options } = this.props` instead? That's the same as `unfilteredOptions` below.